pom.xml

```diff
@@ -360,10 +282,6 @@
                               <version>[${minimum.maven.version},)</version>
                            </requireMavenVersion>
                            <requirePluginVersions>
-                              <unCheckedPlugins>
```

Comment:
maven-eclipse plugin was removed.
versions-maven plugin is not used here and I haven't found it elsewhere either.